tools/scripts/generic.mk

```diff
@@ -207,7 +207,7 @@ endif
 
 #Add to include all directories containing a .h file
 EXTRA_INC_PATHS += $(sort $(foreach dir, $(INCS),$(dir $(dir))))
-CFLAGS += $(addprefix -I,$(EXTRA_INC_PATHS))
+CFLAGS += $(addprefix -I,$(EXTRA_INC_PATHS) $(PLATFORM_INCS))
```

Comment:
I see you defined PLATFORM_INCS in aducm.mk but how about xilinx/altera/stm32 ?

Reply:
They are added directly to CFLAGS and it is ok. In aducm is separate to don't added these directories to the projects because they are already there.